docs/workflow/building/libraries/README.md

```diff
@@ -23,7 +23,7 @@ cd tests
 
 :: Then inner loop build / test
 :: (If using Visual Studio, you might run tests inside it instead)
-pushd ..\src & dotnet build & popd & dotnet build /t:test
+pushd ..\src & dotnet build & popd & dotnet test /t:test
```

Comment:
`dotnet build /t:test` is mentioned in number of other places in the docs. Fix all of them?

Reply:
That change was a mistake and I just reverted it. I updating docs for `dotnet test` with https://github.com/dotnet/runtime/pull/39923.